### PR TITLE
Feature/ #Alert #Badge #Button #Card components. 

### PR DESCRIPTION
**Pull request by SalonSikdar**

### CODE REVIEW

File: pages/components/Card/card.css [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+.mb-3 {
+  margin-bottom: 1rem !important;
+}
+
+.card-header {
+  padding: 0.5rem 1rem;
+  margin-bottom: 0;
+  background-color: rgba(0, 0, 0, 0.03);
+  border-bottom: 1px solid #eeeeee;
```
Comment (anshulraheja) [inline pages/components/Card/card.css]: use variable for colors

File: pages/components/Card/card.css [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+}
+
+.card-body {
+  -ms-flex: 1 1 auto;
```
Comment (anshulraheja) [inline pages/components/Card/card.css]: This is not getting applied. Remove it

File: pages/components/Card/card.css [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+  flex: 1 1 auto;
+  padding: 1rem 1rem;
```
Comment (anshulraheja) [inline pages/components/Card/card.css]: use shorthand for padding
padding: 1rem;

File: pages/components/Card/card.css [hunk 4]
```diff
@@ -0,0 +1,91 @@
+.card {
+  position: relative;
+  display: -ms-flexbox;
+  display: flex;
+  -ms-flex-direction: column;
```
Comment (anshulraheja) [inline pages/components/Card/card.css]: These ms-flex properties are not being applied.

**Review by anshulraheja — APPROVED** [review]
The UI looks great. Kudos for making responsive UI.

File: css/comp-page.css [hunk 5]
```diff
@@ -128,6 +128,9 @@
   line-height: 1.5em;
   color: #464d5f;
```
Comment (iRohitGaur) [inline css/comp-page.css:130]: use variables

File: css/comp-page.css [hunk 6]
```diff
@@ -175,10 +178,11 @@
   text-decoration: none;
   color: #000;
```
Comment (iRohitGaur) [inline css/comp-page.css]: use variables

File: pages/components/Alert/alert.css [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+
+.btn-close:hover {
+  color: #000;
+  text-decoration: none;
+  opacity: 0.75;
+}
+
+.btn-close:focus {
+  outline: 0;
+  box-shadow: 0 0 0 0.25rem rgba(217, 35, 15, 0.25);
```
Comment (iRohitGaur) [inline pages/components/Alert/alert.css]: use variables

**Review by iRohitGaur — APPROVED** [review]
variables are not used for color in many places. merge with updates

File: pages/components/Alert/alert.css [hunk 8]
```diff
@@ -0,0 +1,113 @@
...
+  padding-right: 3rem;
+}
+
+.alert-dismissible .btn-close {
+  position: absolute;
+  top: 0;
+  right: 0;
+  z-index: 2;
+  padding: 1.25rem 1rem;
+}
```
Comment (deekshithmd) [inline pages/components/Alert/alert.css:54]: provide line gap between classes

File: pages/components/Alert/alert.css [hunk 9]
```diff
@@ -0,0 +1,113 @@
...
+.alert-warning {
+  color: var(--primary-color);
+  background-color: #f3d240d8;
```
Comment (deekshithmd) [inline pages/components/Alert/alert.css]: using color variable will be good

File: pages/components/Alert/alert.css [hunk 10]
```diff
@@ -0,0 +1,113 @@
+:root {
+  --transparent-color: #efcb2d00;
+}
+
+.alert {
+  position: relative;
+  padding: 1rem 1rem;
+  margin-bottom: 1rem;
+  border: 1px solid transparent;
```
Comment (sanjitsarkar) [inline pages/components/Alert/alert.css:22]: What's the need of `border: 1px solid transparent;`
as it does not make any difference.

**Review by sanjitsarkar — CHANGES_REQUESTED** [review]

File: pages/components/Card/card.css (hunk 1 above)
Comment (SalonSikdar) [reply pages/components/Card/card.css]: Cool, Done.

File: pages/components/Card/card.css (hunk 2 above)
Comment (SalonSikdar) [reply pages/components/Card/card.css]: inspected, yes. removed

File: pages/components/Card/card.css (hunk 3 above)
Comment (SalonSikdar) [reply pages/components/Card/card.css]: ack, done

File: pages/components/Card/card.css (hunk 4 above)
Comment (SalonSikdar) [reply pages/components/Card/card.css]: Ack, removed

File: css/comp-page.css (hunk 5 above)
Comment (SalonSikdar) [reply css/comp-page.css:130]: Ack, done

File: pages/components/Alert/alert.css (hunk 8 above)
Comment (SalonSikdar) [reply pages/components/Alert/alert.css:54]: Ack, done

File: pages/components/Alert/alert.css (hunk 9 above)
Comment (SalonSikdar) [reply pages/components/Alert/alert.css]: Ack, done

File: pages/components/Alert/alert.css (hunk 10 above)
Comment (SalonSikdar) [reply pages/components/Alert/alert.css:22]: It is used to provide a hairline gap of 1px in the container, Improves the UI

**Pull request merged**